### PR TITLE
De-restrict Price Adjustments

### PR DESCRIPTION
**Pull request by bscarlett**
Order adjustment should not have limits and allow both negative and positive values.

[Asana ref](https://app.asana.com/0/1201314691604883/1205140723282802/f)

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.0.0
+0.0.1
```
Comment (suborbitalbacon) [inline VERSION:1]: First version!! 🎉

File: examples/csv_to_ndjson/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-lexer_schemas @ git+https://github.com/lexerdev/lexer-schemas@0.0.0
+lexer_schemas @ git+https://github.com/lexerdev/lexer-schemas@0.0.1
```
Comment (suborbitalbacon) [inline examples/csv_to_ndjson/requirements.txt:1]: Might be worth mentioning you have to update these in the docs

**Review by suborbitalbacon — APPROVED** [review]
Looks good!

File: examples/csv_to_ndjson/requirements.txt (hunk 2 above)
Comment (bscarlett) [reply examples/csv_to_ndjson/requirements.txt:1]: Yeah, I was actually thinking of a script that'd sync the versions between `VERSION` the README and all example script requirements.

Comment (suborbitalbacon) [reply examples/csv_to_ndjson/requirements.txt:1]: Even better, could make it a pre commit hook as well

**Pull request merged**